src/util/IncognitoBrowser.cpp

```diff
@@ -30,6 +30,10 @@ QString getPrivateSwitch(const QString &browserExecutable)
         {"firefox-esr", "-private-window"},
         {"chromium", "-incognito"},
         {"brave", "-incognito"},
+        {"firefox-devedition", "-private-window"},
```

Comment:
```suggestion
        {"firefox-devedition", "-private-window"},
        {"firefox-developer-edition", "-private-window"},
```

On Arch Linux firefox developer edition uses the name `firefox-developer-edition`.